src/core/components/info.jsx

```diff
@@ -110,7 +110,7 @@ export default class Info extends React.Component {
           <h2 className="title" >{ title }
             { version && <VersionStamp version={version}></VersionStamp> }
           </h2>
-          { host || basePath ? <InfoBasePath host={ host } basePath={ basePath } /> : null }
+          {/* { host || basePath ? <InfoBasePath host={ host } basePath={ basePath } /> : null } */}
```

Comment:
Do we still want this commented out here? If so, can you please add a note as to why it was commented out, and why you'd want to uncomment it?

Reply:
possibly we could re-enable this if we start using a correct base URL, for now this always shows up as localhost. it would require a change in the Goa service that produce the spec.